container/build_tar.py

```diff
@@ -39,6 +39,11 @@
 gflags.DEFINE_string(
     'mode', None, 'Force the mode on the added files (in octal).')
 
+gflags.DEFINE_multistring(
```

Comment:
I'm updating the doc as follows:

    'An empty root directory to add to the layer.  This will create a directory that'
    'is a peer of "root_directory".  "empty_dir" creates an empty directory inside of'
    '"root_directory"'

In Linux layers, the root directory is named '.'  In windows there are two root directories 'Files' (the equivalent of '.') and 'Hives'.  Hives is where the registry deltas would are stored.  The directory needs to be present for the image to load, but there doesn't need to be anything populated.

Reply:
thanks for clarifying. lets try to get this info about 'hives' somewhere in the code docs (like in container/layer.bzl ~ line 88)